app/scripts/lib/constants.js

```diff
@@ -41,6 +41,8 @@ define(function (require, exports, module) {
 
     SYNC_SERVICE: 'sync',
 
+    SYNC11_MIGRATION: 'sync11',
+    AMO_MIGRATION: 'amo',
```

Comment:
@shane-tomlinson should we keep this specific to AMO or is this "generic" oauth email migration?